src/ui/FacetsMobileMode/FacetsMobileMode.ts

```diff
@@ -39,15 +39,25 @@ export class FacetsMobileMode extends Component {
      */
     isModal: ComponentOptions.buildBooleanOption({ defaultValue: false }),
     /**
-     * Whether to display a background behind the facets when the end user expands them in mobile responsive mode.
+     * Whether to display a transparent overlay behind the facets when the end user expands them in mobile responsive mode.
```

Comment:
I find "displaying a transparent overlay" not intuitive. I think, how can I see something that is transparent.

What if we inverted the option? Would it be clearer? e.g `blurBackgroundWhileOpen`

Reply:
translucent overlay? 😂 😂 😂

Reply:
Just `an overlay` ?